sensor/extrinsic_calibration_manager/launch/aip_x1/ground_plane_sensor_kit.launch.xml

```diff
@@ -5,7 +5,6 @@
   <let name="sensor_model" value="aip_x1" />
   <let name="base_frame" value="base_link" />
   <let name="parent_frame" value="sensor_kit_base_link" />
-  <let name="pointcloud_topic" value="/sensing/lidar/top/pointcloud_raw" />
```

Comment:
why remove?

Reply:
Because we are now calibrating base-> lidar for all lidars (velodyne + livox).
That ways we can get the 3D base->lidar calibration for the velodyne and the 6D claibration for the livoxes (3D from the base-lidar and 3D from the 2d ICP)

Reply:
I understood, thanks.